store/rootmulti/store.go

```diff
@@ -757,8 +760,17 @@ func (rs *Store) Snapshot(height uint64, protoWriter protoio.Writer) error {
 	// and the following messages contain a SnapshotNode (i.e. an ExportNode). Store changes
 	// are demarcated by new SnapshotStore items.
 	for _, store := range stores {
+		rs.logger.Debug("Snapshot Begin", "store", store.name)
+
 		exporter, err := store.Export(int64(height))
+		if exporter == nil {
+			rs.logger.Error("Snapshot Failed - exporter nil", "store", store.name)
+			// CV Skip stores that fail to get an exporter
+			//    For example, when iavl/immutable_tree.ndb (nodedb) is nil
+			continue
```

Comment:
There is a bug here.  It is still important to export the empty store for the Merkle root to be correct